src/client/src/orangehrmClaimPlugin/components/AddExpenseModal.vue

```diff
@@ -170,7 +170,7 @@ export default {
         .create({
           expenseTypeId: this.expense.type.id,
           date: this.expense.date,
-          amount: (Math.floor(this.expense.amount * 100) / 100).toFixed(2),
+          amount: this.expense.amount,
```

Comment:
```suggestion
          amount: this.expense.amount.toFixed(2),
```
:thinking: I'm not sure whether we need to add this

Reply:
@RajithaKumara is correct, if number is coming in expense.amount, no need to do these additional calculations. just use `toFixed()`

![image](https://github.com/orangehrm/orangehrm/assets/12880059/047b0c06-82e1-4d44-b097-779f87ce320e)